opencti-platform/opencti-graphql/src/manager/taskManager.js

```diff
@@ -237,6 +238,9 @@ const executeDelete = async (context, user, element, scope) => {
   }
   if (scope === BackgroundTaskScope.Import) {
     await deleteFile(context, user, element.id);
+  } else if (element.entity_type === ENTITY_TYPE_IDENTITY_ORGANIZATION) {
+    // call organizationDelete which will ensure protections (for platform organization & members)
+    await organizationDelete(context, user, element.internal_id);
   } else {
     await deleteElementById(context, user, element.internal_id, element.entity_type);
```

Comment:
wouln't it be better to add the organization check in 'deleteElementById' ? to avoid forgetting to add the check when using it?